includes/class-wc-payments-captured-event-note.php

```diff
@@ -206,7 +206,7 @@ public function get_fee_breakdown() {
 
 		foreach ( $history as $fee ) {
 			$label_type = $fee['type'];
-			if ( isset( $fee['additional_type'] ) ) {
+			if ( isset( $fee['additional_type'] ) && ! empty( $fee['additional_type'] ) ) {
```

Comment:
Thanks for looking into this. May make it simple like this? 

```suggestion
			if ( $fee['additional_type'] ?? '' ) {
```